generative/maisi/data/mask_augmentation_example.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>You can remove the output here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
sure, I removed the cell output.